### PR TITLE
Add `priv_getEeaTransactionCount` API.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## PR description
Add `priv_getEeaTransactionCount` API documentation.

## Fixed Issue(s)
PAN-3256

### CODE REVIEW

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ when processing the private transaction enables greater control over the signing
 Separate private states are maintained for each [privacy group](../../Concepts/Privacy/Privacy-Groups.md) so 
 the account nonce for an account is specific to the privacy group. That is, the nonce for account A for
 privacy group ABC is different to the account nonce for account A for privacy group AB. Use 
-[`priv_getTransactionCount`](../../Reference/API-Methods.md#priv_gettransactioncount) to get 
+[`priv_getTransactionCount`](../../Reference/API-Methods.md#priv_gettransactioncount) or [`priv_getEeaTransactionCount`](../../Reference/API-Methods.md#priv_geteeatransactioncount) to get
```
Comment (macfarla) [inline docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: This is about the line before - 2 out of 3 times it says "account nonce" and the other time just "nonce" - I would be less confused if it was consistent

File: docs/Reference/API-Methods.md [hunk 2]
```diff
@@ -4147,6 +4147,43 @@ Distributes a signed, RLP encoded [private transaction](../HowTo/Send-Transactio
...
+
+!!! important 
+    If sending more than 1 transaction to be mined in the same block (that is, you're not waiting for 
+    the transaction receipt), you must calculate the private transaction nonce outside Besu. 
+
+**Parameters** 
+
+`data` - Account address
+
+`data` - Orion address of the sender
```
Comment (pinges) [inline docs/Reference/API-Methods.md]: Do you want to add that these are base64 encoded. The sender address, as well as the recipient addresses

File: docs/Reference/API-Methods.md [hunk 3]
```diff
@@ -4147,6 +4147,43 @@ Distributes a signed, RLP encoded [private transaction](../HowTo/Send-Transactio
...
+
+`array of data` - Orion addresses of recipients
+
+**Returns** 
+
+`quantity` - Integer representing the number of private transactions sent from the address to the specified privacy group.
```
Comment (MadelineMurray) [inline docs/Reference/API-Methods.md]: Same comment as above. 

privacy group -> group of sender and recipients

File: docs/Reference/API-Methods.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Reference/API-Methods.md]: We probably should

File: docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md (hunk 1 above)
Comment (bgravenorst) [reply docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md]: Done.

File: docs/Reference/API-Methods.md (hunk 2 above)
Comment (bgravenorst) [reply docs/Reference/API-Methods.md]: Done.

File: docs/HowTo/Configure/Configure-HA/High-Availability.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ are sent to multiple nodes, the [`eth_getTransactionCount`](../../../Reference/A
 results can be incorrect. 
 
 !!! note
-    If using [private transactions](../../../Concepts/Privacy/Privacy-Overview.md), `priv_getTransactionCount` is used to obtain 
+    If using [private transactions](../../../Concepts/Privacy/Privacy-Overview.md), [`priv_getTransactionCount`](../../Reference/API-Methods.md#priv_gettransactioncount) or [`priv_getEeaTransactionCount`](../../Reference/API-Methods.md#priv_geteeatransactioncount) is used to obtain
```
[the remark this answers is not in the document]
Comment (bgravenorst) [reply docs/HowTo/Configure/Configure-HA/High-Availability.md]: Done.

File: docs/Reference/API-Methods.md (hunk 3 above)
Comment (bgravenorst) [reply docs/Reference/API-Methods.md]: Done

**Pull request merged**